<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encryption Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
    <item>
     <widget class="QSpinBox" name="shiftSpinBox">
      <property name="maximum">
       <number>25</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="keyLineEdit"/>
    </item>
    <item>
     <widget class="QRadioButton" name="caesarRadioButton">
      <property name="text">
       <string>Caesar Cipher</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="vigenereRadioButton">
      <property name="text">
       <string>Vigenere Cipher</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rsaRadioButton">
      <property name="text">
       <string>RSA Encryption</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="aesRadioButton">
      <property name="text">
       <string>AES Encryption</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rc4RadioButton">
      <property name="text">
       <string>RC4 Encryption</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="encryptButton">
      <property name="text">
       <string>Encrypt</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="decryptButton">
      <property name="text">
       <string>Decrypt</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="cipherTextEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
